README.md

```diff
@@ -2,51 +2,58 @@
 
 Moxin is an AI LLM client written in Rust, and demonstrates the power of the [Makepad UI toolkit](https://github.com/makepad/makepad) and [Project Robius](https://github.com/project-robius), a framework for multi-platform application development in Rust.
 
-> ⚠️ Moxin is in beta. Please [file an issue](https://github.com/moxin-org/moxin/issues/new) if you encounter bugs or unexpected results.
+> ⚠️ Moxin is in beta. Please [file an issue](https://github.com/moxin-org/moly/issues/new) if you encounter bugs or unexpected results.
 
 The following table shows which host systems can currently be used to build Moxin for which target platforms.
-| Host OS | Target Platform | Builds? | Runs? | Packaging Support                            |
+| Host OS | Target Platform | Builds? | Runs? | Packaging Support |
 | ------- | --------------- | ------- | ----- | -------------------------------------------- |
-| macOS   | macOS           | ✅      | ✅    | `.app`, [`.dmg`]                             |
-| Linux   | Linux           | ✅      | ✅    | [`.deb` (Debian dpkg)], [AppImage], [pacman] |
-| Windows | Windows (10+)   | ✅      | ✅    | `.exe` (NSIS)                                |
+| macOS | macOS | ✅ | ✅ | `.app`, [`.dmg`] |
+| Linux | Linux | ✅ | ✅ | [`.deb` (Debian dpkg)], [AppImage], [pacman] |
+| Windows | Windows (10+) | ✅ | ✅ | `.exe` (NSIS) |
```

Comment:
why discard the nice formatting here? 😄

Reply:
I had to fight a bit against the prettier formatter in my VSC, but yeah.. I like this human-formatted version which is more readable 👍